### ISSUE TITLE
Cannot get page content (but have workaround)

### ISSUE THREAD

**Issue by rgardler** [microsoftgraph/msgraph-sdk-javascript#55]
After having retrieved a list of pages I am attempting to get the content of that page. I am using, the response does not include the content, this can be reproduced as follows

```javascript
getGraphClient().api('/me/onenote/pages/' + entry.pageId + '/content')
                    .get((err, res, raw) => {
                        if (err) {
                            console.log("ERROR: Error retrieving body for page with id " + entry.pageId);
                            console.log(err)
                            entry.body = err.message;
                        } else {
                            console.log(res)
                            console.log(raw.res.text); 
                        }
                    });)
```

`console.log(res)` displays '{}'

WORKAROUND: `console.log(raw.res.text)` provides the page content.

**Comment by MIchaelMainer** [comment]
I think this is by design. The content comes back as HTML, not as JSON, and therefore isn't deserialized into a JS object.

**Comment by rgardler** [comment]
OK, Then at least have the SDK behave in a way that is predictable. Returning an empty object in the place every other call that I've tried provides the actual content is non-intuitive. I spent hours debugging this, in the end I had to go into the SDK code to figure it out. 

At the very least I suggest you actually document this so that other people don't hit this problem. The obvious place is https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/page_get
